docs/server/source/production-deployment-template/troubleshoot.rst

```diff
@@ -104,3 +104,28 @@ MongoDB instance correctly.
 
 More information about this configuration is provided in
 :doc:`this document <cloud-manager>`.
+
+6. Create a Persistent Volume from existing Azure disk storage Resource
+---------------------------------------------------------------------------
+The configuration file to create PV for Mongo Stateful Set is located at ``mongodb/mongo-pv.yaml``.
+Set the ``name``, ``diskName`` and ``diskURI`` values and run the following command to create PVs
+required for MongoDB Stateful Set.
```

Comment:
To be friendly, you could remind them that there are two PVs to be created: one for the MongoDB data and one for the MongoDB config values.